nixos/modules/services/security/nginx-sso.nix

```diff
@@ -9,6 +9,9 @@ let
   cfg = config.services.nginx.sso;
   format = pkgs.formats.yaml { };
   configPath = "/var/lib/nginx-sso/config.yaml";
```

Comment:
Not directly related to your change, but this should probably be changed to `/run/nginx-sso/config.yaml`. Otherwise, the secret values will be stored on disk in that file

Reply:
Will open a new issue for this.